lib/core.ps1

```diff
@@ -285,7 +285,47 @@ function isFileLocked([string]$path) {
 }
 
 function is_directory([String] $path) {
-    return (Test-Path $path) -and (Get-Item $path) -is [System.IO.DirectoryInfo]
+    return (Test-Path $path) -and (Get-Item -Path $path) -is [System.IO.DirectoryInfo]
+}
+
+function is_junction([String] $path) {
+    return (Test-Path $path) -and (Get-Item -Path $path).LinkType -eq 'Junction'
+}
+
+function create_junction([String] $link, [String] $target) {
+    if (!(Test-Path $link) -and (is_directory $target)) {
+        New-Item -ItemType Junction -Path (Split-Path -Path $link) -Name (Split-Path -Leaf $link) -Target $target | Out-Null
+        $dirInfo = New-Object System.IO.DirectoryInfo($link)
+        $dirInfo.Attributes = $dirInfo.Attributes -bor [System.IO.FileAttributes]::ReadOnly
+        return $true
+    }
+    return $false
+}
+
+function remove_junction([String] $link) {
+    if(is_junction $link) {
+        $dirInfo = New-Object System.IO.DirectoryInfo($link)
+        $dirInfo.Attributes = $dirInfo.Attributes -band (-bnot [System.IO.FileAttributes]::ReadOnly)
+        $dirInfo.Delete()
```

Comment:
Remove-Item xxx -Recurse -Force could remove read-only junction. So if we want to delete junction or hardlink, a single function that Remove-Item xxx -Recurse -Force is enough.

Reply:
Yes, but I got an error message saying powershell is not in interactive mode, so I changed it to a simple delete be the object is already there.

_Sent from my OnePlus5T using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
I've tested with changing two `remove_` function's contents to `Remove-Item` (after fix type in L1216), and also with replacing all `remove_junction` and `remove_hardlink` with `Remove-Item xxx -Recurse -Force | Out-Null`, and gotten no error. Maybe need more test.

PS. I've googled that without `-Force` may return the error that `not in interactive mode`. May help.